packages/create-workleap/spinner.ts

```diff
@@ -0,0 +1,40 @@
+import { spinner } from "@clack/prompts";
```

Comment:
After re-reviewing, is a Spinner wrapper really necessary? it feels like it add complexity for nothing? 

Instead of doing 
```
  StartSpinner();
  await CallFoundryAsync(outputDirectory, template, scope);
  StopSpinner();
```

We could simply have
```
import { spinner } from "@clack/prompts";
...
const loader = spinner("Running...");
loader.start();
await CallFoundryAsync(outputDirectory, template, scope);
loader.stop();
```


Is the spinner doing something else i'm unaware of?

Reply:
Not really... indeed, we can simply not wrap it up now.

It has more sense previously when it was an object and can have multiple instances. And for some weird reason I was with the idea that the `start` and `stop` text has to be the same, but after a quick test it's not required.